tools/setup

```diff
@@ -8,7 +8,7 @@ mkdir -p ${script_dir}/build
 cd ${script_dir}/build
 
 # Setup cmake
-cmake -DCMAKE_BUILD_TYPE=Release \
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo \
```

Comment:
the same as default -> Release, check where this is called, and pass value if possible